src/table/TableBodyWrapper.jsx

```diff
@@ -58,6 +60,8 @@ function TableBodyWrapper({
     colIdx: -1,
     isEnabled: selectionsEnabled,
   });
+  const [srNotation, setSrNotation] = useState('');
+  const MAX_NOTATION_DELAY = 250;
```

Comment:
Did you try without the delay?

Reply:
Yes, and the result was not really what we were expected. 

### here was my idea: 
Since react batches the `setState` updates in lifecycle methods and (synthetic) event handlers, this will be the case in our current situation if we try to remove the `setTimeout`, and in result, we would get the name of the cell only from screen reader ( witch is the default behaviour of it ) since the label would **not** update in subsequent row names that has the same value ( like two or couple of rows below together with same value, for example imagine row [6,0] = Sweden, [7,0]= Sweden, [8,0] = Sweden, ... ) and this will happen and happen until we get to another cell that has a different value.

### here was the result: 
This was the exact behaviour that i get from my test result in windows machine without `setTimeout`. and since the label does not updated because of the batched state updates, which would result in same label element value, i did not get NVDA screen reader to pick up the changes.

So what is the escape case from this situation? there are two: 
1) set timeout and 
2) ajax calls 
which both of them are not handled by react, it's JS related behaviour and API. 

In our case i'm using the setTimeout to escape out this behaviour of state update batch of react. I think the change we could do here is to do a favour for react and put the functionality at the end of event queue by setting the time to `0` in our setTimeout.

Reply:
Isn't this because the information is put into a separate label? Should the information not be present on every single cell instead?

Reply:
this way was our first implementation and since if we use `aria-live="assertive"`, that looked a bit unnecessary to put any other information to cell except the cell content; because we update the label and screen reader picks it up.  

even by shoving more info into cell content, we still going to need setTimeout here, because we need to trigger the screen reader to pick up the changes from label.